File: apprefurbished/src/main/java/com/androidcollider/easyfin/common/managers/format/date/DateFormatModule.java
package com.androidcollider.easyfin.common.managers.format.date;

import dagger.Module;
import dagger.Provides;

/**
 * @author Ihor Bilous
 */

@Module
public class DateFormatModule {

    @Provides
    DateFormatManager provideDateProvideManager() {
        return new DateFormatManager();
    }
}

File: apprefurbished/src/main/java/com/androidcollider/easyfin/common/managers/format/date/DateFormatModule.kt
package com.androidcollider.easyfin.common.managers.format.date

import dagger.Module
import dagger.Provides

/**
 * @author Ihor Bilous
 */
@Module
class DateFormatModule {
    @Provides
    fun provideDateProvideManager(): DateFormatManager {
        return DateFormatManager()
    }
}